modules/openapi-generator/src/main/resources/JavaSpring/pojo.mustache

```diff
@@ -146,15 +135,20 @@ public class {{classname}} {{#parent}}extends {{{.}}}{{/parent}}{{^parent}}{{#ha
   {{#swagger1AnnotationLibrary}}
   @ApiModelProperty({{#example}}example = "{{{.}}}", {{/example}}{{#required}}required = {{required}}, {{/required}}{{#isReadOnly}}readOnly = {{{isReadOnly}}}, {{/isReadOnly}}value = "{{{description}}}")
   {{/swagger1AnnotationLibrary}}
-  public {{>nullableDataType}} {{getter}}() {
+  public {{>dataType}} {{getter}}() {
     return {{name}};
   }
 
   {{#vendorExtensions.x-setter-extra-annotation}}
   {{{vendorExtensions.x-setter-extra-annotation}}}
   {{/vendorExtensions.x-setter-extra-annotation}}
-  public void {{setter}}({{>nullableDataType}} {{name}}) {
+  public void {{setter}}({{{datatypeWithEnum}}} {{name}}) {
```

Comment:
Previous discussion was to have the setter take in the same as the data type declaration - `Optional<T>` or `JsonNullable<T>` instead of just taking in `T`.

I'm not sure if this will work with the Jackson JDK8 module.

So I think this should be `{{dataType}}`